app/src/main/java/rick_and_morty/ui/episodes/composables/EpisodesRow.kt

```diff
@@ -0,0 +1,67 @@
+package rick_and_morty.ui.episodes.composables
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Color.Companion.Gray
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.rememberImagePainter
+import rick_and_morty.data.model.CharacterResultsDto
+import rick_and_morty.data.model.episodes.EpisodesResultDto
+
+@Composable
+fun EpisodesRow(
+    episodeResultsDto: EpisodesResultDto,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(180.dp)
+            .border(border = BorderStroke(width = 1.dp, color = Color.White))
+            .clickable (onClick = {}),
```

Comment:
Why adding this when there will be no action triggered?

Reply:
Removed